cores/esp8266/time.cpp

```diff
@@ -133,21 +133,58 @@ int _gettimeofday_r(struct _reent* unused, struct timeval *tp, void *tzp)
 
 void configTime(int timezone_sec, int daylightOffset_sec, const char* server1, const char* server2, const char* server3)
 {
-    char tzstr [64];
-
     // There is no way to tell when DST starts or stop with this API
     // So DST is always integrated in TZ
     // The other API should be preferred
 
+#if 0
```

Comment:
Instead of an #if #else I suggest a comment block along the lines of: "the following is a hack of the internal structures of libc because... (insert explanation)... It is equivalent to the following code: (insert code currently inside the #if 0).